src/redux/news/news.sagas.js

```diff
@@ -89,7 +89,7 @@ function* handleArticleLoad({ payload }) {
   }
 }
 
-function* handleNewsError(e) {
+export function* handleNewsError(e) {
```

Comment:
Why have you added  `export` here? This saga function should be used only inside this file and root news saga

Reply:
I exported it for tests